app/models/vote.rb

```diff
@@ -0,0 +1,11 @@
+class Vote < ApplicationRecord
+  belongs_to :user
+  belongs_to :work
```

Comment:
You can use the `dependent: :destroy` property to allow your votes to be deleted alongside your Works. That way you don't get stuck with Works you can never delete.